docs/gettsim_objects/input_variables.rst

```diff
@@ -32,9 +32,9 @@ household.
 +---------------------------+---------------------------------------------+--------------+
 | _`in_ausbildung`          | Dummy: Employment status “in education”     | bool         |
 +---------------------------+---------------------------------------------+--------------+
-| _`selbstständig`          | Dummy: Individual is self-employed          | bool         |
+| _`selbstständig`          | Dummy: Full-time self-employed              | bool         |
```

Comment:
Why would it have to be full time? I guess it is more about the main activity? Will need to rethink that classification at some point, anyhow.

Reply:
You are right. We need the translation of "hauptberuflich". 

How about `Dummy: Self-employed (main profession)` ? Alternatively, we can just add the German term

Reply:
Just use that, perfectly fine. 

Before a 1.0 release we should definitely go for a categorical variable with the main type of activity (employment subject to social security contributions, civil servant, self-employed, unemployed, out of the labor force, parental leave, pensioner,  ...). Will get hairy with pensioner / employment definition from 2023 onwards, of course... Don't we already have an issue for that?

Reply:
👍 

I don't remember and haven't found one.